miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/errors/MiniAppSecureStorageError.kt

```diff
@@ -10,42 +11,42 @@ internal class MiniAppSecureStorageError(val type: String? = null, val message:
     MiniAppBridgeError(type, message) {
 
     companion object {
-        private const val SecureStorageFullError = "SecureStorageFullError"
-        private const val SecureStorageIOError = "SecureStorageIOError"
-        private const val SecureStorageUnavailableError = "SecureStorageUnavailableError"
-        private const val SecureStorageBusyError = "SecureStorageBusyError"
+        private const val SecureStorageNoSpaceAvailableError = "SecureStorageNoSpaceAvailableError"
+        private const val SecureStorageInsertItemsFailedError = "SecureStorageInsertItemsFailedError"
+        private const val SecureStorageDeleteItemsFailedError = "SecureStorageDeleteItemsFailedError"
+        private const val SecureStorageFatalDatabaseRuntimeError = "SecureStorageFatalDatabaseRuntimeError"
```

Comment:
@gopi-karmakar , We need to be aligned with error types defined in JS Bridge. [Please check here](https://github.com/rakutentech/js-miniapp/blob/master/js-miniapp-bridge/src/types/error-types/secure-storage-errors.ts)

Reply:
@rleojoseph But I think we should let the `MiniApps` or the `HostApp` developers the correct error cause or reason like, `Insert Items Failed ` or `Delete Items Failed` or you meant we should just throw the `SecureStorageIOError` in any of those scenarios with the proper error message only? That seems to be fine too.

Reply:
@gopi-karmakar, I certainly agree. But it's better to have these errors aligned on all 3 platforms. I think we had a discussion regarding this in our tech discussion meeting and decided to proceed further with those errors. You can also check the Tech spec documentation https://confluence.rakuten-it.com/confluence/display/ONEAPP/Secure+Storage+for+Sensitive+Data
We need to update them anyway.

Reply:
However, Added the tracking of the DB state and addressed the above concern. Now returning the same errors as per the DB status. Already tested. Thanks.